readme.md

```diff
@@ -7,35 +7,6 @@
 This project is a fork of the Ms Office EWS api
 There are a few code changes.
 
-#### Adding Dependencies
-To depend using maven:
-
-1. Go to https://jitpack.io and sign in with github (top right of the screen). You'll have to grant jitpack authorization
-to do a few things (e.g., access private repos).
-
-2. After you're logged in, go <a href="https://jitpack.io/private#auth">here</a> and click the "maven" tab. Copy the xml
-into your ~/.m2/settings.xml file.
-
-3. In the pom where you need to depend on a stitch module, add the jitpack repo to your repositories section:
-
-  ```xml
-  <repository>
-      <id>jitpack.io</id>
-      <url>https://jitpack.io</url>
-  </repository>
-  ```
-
-4. Now, you can depend on the ews-java-api module by using `com.github.SpanningCloudApps` as the `groupId` and `ews-java-api`
-as the `artifactId`. For `version`, use `tagName` to depend on the a tag/release.
-
-  ```xml
-  <dependency>
-      <groupId>com.github.SpanningCloudApps</groupId>
-      <artifactId>ews-java-api</artifactId>
-      <version>2.1.2</version>
-  </dependency>
-  ```
-
```

Comment:
Removed these Spanning specific instructions. The Cloudbees stuff is gone now too.